columnflow/tasks/framework/remote.py

```diff
@@ -513,12 +567,17 @@ def htcondor_use_local_scheduler(self):
         # remote jobs should not communicate with ther central scheduler but with a local one
         return True
 
+    def htcondor_destination_info(self, info: dict[str, str]) -> dict[str, str]:
+        info = super().htcondor_destination_info(info)
+        info = self.common_destination_info(info)
+        return info
+
 
 _default_slurm_flavor = law.config.get_expanded("analysis", "slurm_flavor", "maxwell")
 _default_slurm_partition = law.config.get_expanded("analysis", "slurm_partition", "cms-uhh")
 
 
-class SlurmWorkflow(AnalysisTask, law.slurm.SlurmWorkflow):
+class SlurmWorkflow(AnalysisTask, law.slurm.SlurmWorkflow, RemoteWorkflowMixin):
```

Comment:
Does it make sense to also use the `RemoteWorkflowMixin.add_bundle_requirements` function in the `slurm_workflow_requires`?

Reply:
Slurm is designed in a way that jobs are actually executed inside a directory local to the submission node, with all software and env variables available on the remote side. Thus, there was no mechanism in place yet for the bundling of repos / sandboxes, and I left this part to a future PR in case this becomes necessary for Slurm jobs.